neo/io/basefromrawio.py

```diff
@@ -107,6 +107,9 @@ def read_block(self, block_index=0, lazy=False, signal_group_mode=None,
 
         if signal_group_mode is None:
             signal_group_mode = self._prefered_signal_group_mode
+            if self._prefered_signal_group_mode == 'split-all':
+                self.logger.warning("the default signal_group_mode will'\
+                        'be 'split-all' in next release")
```

Comment:
shouldn't this read `he default signal_group_mode will be 'group_by_same_units' in next release`?

Reply:
yes of course.
I totally stupid I wrote the contrary.